presentation/src/main/java/com/anytypeio/anytype/presentation/extension/AnalyticsExt.kt

```diff
@@ -1316,16 +1316,23 @@ fun CoroutineScope.sendChangeWidgetSourceEvent(
         eventName = EventsDictionary.changeWidgetSource,
         props = Props(
             buildMap {
-                when(view) {
+                when (view) {
                     BundledWidgetSourceView.Collections -> {
                         put(WidgetAnalytics.TYPE, WidgetAnalytics.WIDGET_SOURCE_COLLECTIONS)
                     }
+
                     BundledWidgetSourceView.Favorites -> {
                         put(WidgetAnalytics.TYPE, WidgetAnalytics.WIDGET_SOURCE_FAVORITES)
                     }
+
                     BundledWidgetSourceView.Recent -> {
                         put(WidgetAnalytics.TYPE, WidgetAnalytics.WIDGET_SOURCE_RECENT)
                     }
+
+                    BundledWidgetSourceView.RecentLocal -> {
+                        put(WidgetAnalytics.TYPE, WidgetAnalytics.WIDGET_SOURCE_RECENT)
```

Comment:
looks like there should be something like WidgetAnalytics.WIDGET_SOURCE_RECENT_LOCAL